src/ucs/memory/memory_type.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2019.  ALL RIGHTS RESERVED.
```

Comment:
`2019`?

Reply:
Probably we need some script that will bump up current date for every single copyright out there,